artemis-console-extension/artemis-extension/packages/artemis-console-plugin/src/queues/QueuesView.tsx

```diff
@@ -112,9 +114,9 @@ export const QueuesView: React.FunctionComponent<Navigate> = navigate => {
           <TextContent>
             <Text component={TextVariants.h1}>Using Filter Expressions</Text>
             <Text component={TextVariants.h2}>Apache ActiveMQ Artemis provides a powerful filter language based on a subset of the SQL 92 expression syntax.</Text>
-            <Text>It is the same as the syntax used for JMS & Jakarta Messaging selectors. 
-              For documentation on JMS selector syntax please the JavaDoc for javax.jms.Message or jakarta.jms.Message</Text>
-              <Text>A Filter will search for messages that have a matching header.</Text>
+            <Text>It is similar to the syntax used for JMS Message selectors. 
+              For documentation on JMS selector syntax please the JavaDoc for javax.jms.Message or <Link target='_BLANK' to={'https://jakarta.ee/specifications/messaging/3.1/apidocs/jakarta.messaging/jakarta/jms/message'}>jakarta.jms.Message</Link></Text>
+              <Text>A Filter will search for messages that have a matching property.</Text>
```

Comment:
It was more the 50+ instances of 'header' below that I meant, but I don't feel strongly about it if noone else thinks its an issue, so will merge this as-is.